### PR TITLE
[sknpm] Do not copy typescript files

### PR DESCRIPTION
**Pull request by skiplabsdaniel**
The typescript files are no longer copied, so in case of error, the correct file path is displayed.

### CODE REVIEW

File: sknpm/src/utils.sk [hunk 1]
```diff
@@ -77,29 +77,36 @@ fun relative(path: String, to: String): ?String {
...
+        "incremental" => JSON.Bool(true),
+        "verbatimModuleSyntax" => JSON.Bool(true),
+      ],
+    ),
+  ];
+  configMap.set(
+    "include",
+    JSON.Array(Vector::createFromItems(deps.map(d -> JSON.String(d.i1)))),
+  );
+  config = JSON.Object(configMap.chill());
```
Comment (mbouaziz) [inline sknpm/src/utils.sk]: Nit: why not `"include" => ...`, avoid the `mutable` and the `chill`?

File: sql/ts/tests/service_server.mk [hunk 2]
```diff
@@ -15,29 +15,31 @@ SKDB_CLI=$(SRV_DIR)/node_modules/skdb/dist/skdb-cli.mjs
...
 SKGW_JAR=$(SKGW_DIR)/build/libs/dev.jar
-SERVER_JAR=$(SRV_DIR)/server.jar
+SERVER_JAR=$(BUILD_DIR)/server.jar
+CONF_FILE=$(BUILD_DIR)/skdb.conf
 
 default: build
 
 .PHONY: build_server
 build_server:
-	@source $(SDKMAN_INIT) && cd $(SERVER_DIR) && gradle --console plain build 1>&2 > $(SKDB_DATABASES)/build_server.log
```
Comment (jberdine) [inline sql/ts/tests/service_server.mk:28]: I don't understand why we don't need this any more

File: sql/ts/tests/service_server.mk [hunk 3]
```diff
@@ -15,29 +15,31 @@ SKDB_CLI=$(SRV_DIR)/node_modules/skdb/dist/skdb-cli.mjs
...
+	$(SRV_DIR)/service_server.sh $(SDKMAN_DIR) $(SKGW_DIR) $(CONF_FILE) $(SKDB_DATABASES) $(SERVER_JAR)
 	@echo sknpm.env:SKDB_CLI=$(SRV_DIR)/node_modules/skdb/dist/skdb-cli.mjs
 
-.PHONY: $(SRV_DIR)/skdb.conf
-$(SRV_DIR)/skdb.conf: $(SKDB_CLI) $(SKDB_DATABASES) $(SKDB_INIT)
-	@echo "skdb_port=8110" > $(SRV_DIR)/skdb.conf
-	@echo "skdb_databases=$(SKDB_DATABASES)" >> $(SRV_DIR)/skdb.conf
-	@echo "skdb=$(SKDB_CMD)" >> $(SRV_DIR)/skdb.conf
-	@echo "skdb_init=$(SKDB_INIT)" >> $(SRV_DIR)/skdb.conf
-	@echo "skdb_add_cred=cd $(SKDB_CLI_DIR) && node $(SKDB_CLI_CMD) --add-cred --host ws://localhost:%d --db %s --access-key %s <<< \"%s\"" >> $(SRV_DIR)/skdb.conf
```
Comment (jberdine) [inline sql/ts/tests/service_server.mk:43]: Here also, why is this not needed

File: sql/ts/tests/service_server.mk (hunk 2 above)
Comment (skiplabsdaniel) [reply sql/ts/tests/service_server.mk:28]: Because it's managed by the following line.

File: sql/ts/tests/service_server.mk (hunk 3 above)
Comment (skiplabsdaniel) [reply sql/ts/tests/service_server.mk:43]: skdb_add_cred is no more manage by the dev server.

**Pull request closed without merging**